GitPrompt.ps1

```diff
@@ -105,12 +105,66 @@ if (Get-Module NuGet) {
     $WindowTitleSupported = $false
 }
 
+# Color codes from https://msdn.microsoft.com/en-us/library/windows/desktop/mt638032(v=vs.85).aspx
+function Get-ForegroundVirtualTerminalSequence($Color) {
+    $e = [char]27 + "["
+    switch ($Color) {
+      ([ConsoleColor]::Black)       { "${e}30m" }

```

Comment:
Actually, using an array would be more efficient - indexing into the array will be much faster.